### PR TITLE
Update Mail subject and replyTo by message

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

**Comment by ottaviano** [comment]
[ch8099]

File: app/config/services/services.xml [hunk 1]
```diff
@@ -596,6 +595,12 @@
             <argument type="string">%env(MAILCHIMP_MEMBER_INTEREST_INTEREST_GROUP_ID)%</argument>
             <argument type="string">%env(MAILCHIMP_SUBSCRIPTION_TYPE_INTEREST_GROUP_ID)%</argument>
         </service>
+        <service id="AppBundle\Mailchimp\Campaign\Listener\SetCampaignReplyToListener">
+            <tag name="kernel.event_subscriber"/>
```
Comment (thomasbisignani) [inline app/config/services/services.xml]: Why adding manually this tag ? the `autoconfigure` is enabled.

File: app/config/services/services.xml [hunk 2]
```diff
@@ -596,6 +595,12 @@
...
+        </service>
+        <service id="AppBundle\Mailchimp\Campaign\Listener\UpdateCampaignSubjectListener">
+            <tag name="kernel.event_subscriber"/>
```
Comment (thomasbisignani) [inline app/config/services/services.xml]: Same here, thanks to the `autoconfigure`, you can remove this line.

File: src/Mailchimp/Campaign/Listener/SetCampaignReplyToListener.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AppBundle\Mailchimp\Campaign\Listener;
+
+use AppBundle\AdherentMessage\AdherentMessageTypeEnum;
+use AppBundle\Mailchimp\Events;
+use AppBundle\Mailchimp\RequestEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class SetCampaignReplyToListener implements EventSubscriberInterface
```
Comment (thomasbisignani) [inline src/Mailchimp/Campaign/Listener/SetCampaignReplyToListener.php]: I know in the project, a lot of subscribers are suffixed by `Listener`.

What do you think to prefix now all new event subscribers  by `Subscriber` ?

File: src/Mailchimp/Campaign/Listener/UpdateCampaignSubjectListener.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace AppBundle\Mailchimp\Campaign\Listener;
+
+use AppBundle\AdherentMessage\AdherentMessageTypeEnum;
+use AppBundle\Mailchimp\Events;
+use AppBundle\Mailchimp\RequestEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class UpdateCampaignSubjectListener implements EventSubscriberInterface
```
Comment (thomasbisignani) [inline src/Mailchimp/Campaign/Listener/UpdateCampaignSubjectListener.php]: Same here for the prefix.

File: app/config/services/services.xml (hunk 1 above)
Comment (ottaviano) [reply app/config/services/services.xml]: oh yes, fixed, thanks

File: src/Mailchimp/Campaign/Listener/SetCampaignReplyToListener.php (hunk 3 above)
Comment (ottaviano) [reply src/Mailchimp/Campaign/Listener/SetCampaignReplyToListener.php]: why not, done

File: src/Mailchimp/Campaign/Listener/UpdateCampaignSubjectListener.php (hunk 4 above)
Comment (ottaviano) [reply src/Mailchimp/Campaign/Listener/UpdateCampaignSubjectListener.php]: done

File: app/config/services/services.xml (hunk 2 above)
Comment (ottaviano) [reply app/config/services/services.xml]: fixed thank you

**Pull request merged**